<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemStatus</class>
 <widget class="QWidget" name="SystemStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>系统状态</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	font-family:微软雅黑;
	font-size:15px;
	color: #1d649c;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="4,4,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>共识时间</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PlotWidget" name="guiplot_consensus" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>生长速率</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="PlotWidget" name="guiplot_growthRate" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>统计</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,6">
      <item>
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>防篡改率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rate_anti">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
